code/game/machinery/excelsior/ex_teleporter.dm

```diff
@@ -1,5 +1,8 @@
+var/list/global/excelsior_teleporters = list() //This list is used to make turrets more efficient
+
 /obj/machinery/complant_teleporter
 	name = "excelsior long-range teleporter"
+	desc = "a powerful one way teleporter that allows shipping in construction materials. Takes a long time to charge"
```

Comment:
"A powerful one-way receiving teleporter that allows shipping in construction materials. Takes a long time to charge."